src/schemes/fluid/entropically_damped_sph/system.jl

```diff
@@ -62,7 +63,10 @@ struct EntropicallyDampedSPHSystem{NDIMS, ELTYPE <: Real, IC, M, DC, K, V,
                                          alpha=0.5, viscosity=nothing,
                                          acceleration=ntuple(_ -> 0.0,
                                                              ndims(smoothing_kernel)),
-                                         source_terms=nothing)
+                                         source_terms=nothing, buffer=nothing)
+        (buffer ≠ nothing) && (buffer = SystemBuffer(nparticles(initial_condition), buffer))
```

Comment:
Wait, apparently I misunderstood something here.
I thought the buffer is just to mark particles in the system as active and inactive. Therefore, shouldn't the buffer size always be exactly the same as the system size?

Reply:
> Therefore, shouldn't the buffer size always be exactly the same as the system size?

Yes, but initially the `InitialCondition` defines the size of the system. By passing a buffer size, one can increase the `InitialCondition` by recreating/reallocating them and thus increase the system size with inactive particles. The problem is, that I can't just `resize!` the `InitialCOndition`s since arrays are used.